src/pages/wiki/[slug]/index.tsx

```diff
@@ -99,9 +128,46 @@ const Wiki = () => {
                     justify="space-between"
                     direction={{ base: 'column', md: 'row' }}
                   >
-                    <WikiMainContent wiki={wiki} />
-                    <WikiInsights wiki={wiki} />
+                    <Flex
+                      display={{ lg: 'flex', base: 'none', md: 'flex' }}
+                      flexDirection={{
+                        base: 'column-reverse',
+                        lg: 'row',
+                        md: 'row',
+                      }}
+                    >
+                      <WikiMainContent wiki={wiki} mobileView={false} />
+                      <WikiInsights wiki={wiki} />
+                    </Flex>
+                    <Flex
+                      display={{ lg: 'none', base: 'block', md: 'none' }}
+                      flexDirection={{
+                        base: 'column-reverse',
+                        lg: 'row',
+                        md: 'row',
+                      }}
+                    >
+                      <Heading mb={2} p={4}>
+                        {wiki?.title}
+                      </Heading>
+                      <WikiInsights wiki={wiki} />
+                      <WikiMainContent wiki={wiki} mobileView />
+                    </Flex>
                   </Flex>
+                  {wiki.media && wiki.media.length > 0 && (
+                    <RelatedMediaGrid media={wiki.media} />
+                  )}
+                  <Flex
+                    display={{ base: 'block', lg: 'none', md: 'none' }}
+                    flexDirection="column"
+                  >
+                    <MobileMeta
+                      metadata={wiki.metadata}
```

Comment:
If the type defined gets disturbing, you can do
```suggestion
                      metadata={wiki.metadata as Meta[]}
```